scripts/pylint_custom_plugin/pylint_guidelines_checker.py

```diff
@@ -1705,6 +1705,74 @@ def visit_functiondef(self, node):
     # this line makes it work for async functions
     visit_asyncfunctiondef = visit_functiondef
 
+class CheckApiVersion(BaseChecker):
+    __implements__ = IAstroidChecker
+
+    name = "check-api-version"
+    priority = -1
+    msgs = {
+        "C4748": (
+            "Check api version",
```

Comment:
I think we can be more descriptive on the linting error message here 😄 and maybe include the guideline link?